.circleci/config.yml

```diff
@@ -6,16 +6,16 @@ orbs:
 parameters:
   image_suffix:
     type: string
-    default: '-vbab548a'
+    default: '-dev-30a9ffe'
```

Comment:
Reminder to not forget to change this to non dev image once the the-process PR is merged.